src/v2/view-builder/views/PollView.js

```diff
@@ -43,12 +46,13 @@ const Body = BaseForm.extend(Object.assign(
 
     remove() {
       BaseForm.prototype.remove.apply(this, arguments);
-      this.stopPolling();
+      clearTimeout(this.refreshTimeout);
     },
 
     triggerAfterError() {
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
-      this.stopPolling();
+      clearTimeout(this.refreshTimeout);
+      this.stopCountDown();
```

Comment:
maybe move `stop/start CountDown` to this view and remove line 60 (compose `polling.js`)